src/routes/docs/quick-starts/kotlin/+page.markdoc

```diff
@@ -0,0 +1,173 @@
+Open you `build.gradle.kts` file and implement the following dependency.
+
+```groovy
+dependencies {
+    ... other dependencies
+    implementation("io.appwrite:sdk-for-kotlin:4.1.0")
+}
+```
+
+{% /section %}
+{% section #step-4 step=4 title="Import Appwrite" %}
+
+Find your project ID in the **Settings** page. Also, click on the **View API Keys** button to find the API key that was created earlier. 
+
+{% only_dark %}
+![Project settings screen](/images/docs/quick-starts/dark/project-id.png)
+{% /only_dark %}
+{% only_light %}
+![Project settings screen](/images/docs/quick-starts/project-id.png)
+{% /only_light %}
+
+Open the file `Main.kt` and initialize the Appwrite Client. Replace `<YOUR_PROJECT_ID>` with your project ID and `<YOUR_API_KEY>` with your API key.
+
+```kotlin
+import io.appwrite.Client
+import io.appwrite.services.Databases
+import io.appwrite.models.Database as DatabaseModel
+import io.appwrite.models.Collection
+import kotlinx.coroutines.runBlocking
+import java.util.UUID
+val client = Client()
+    .SetEndpoint("https://cloud.appwrite.io/v1")
+    .SetProject("<YOUR_PROJECT_ID>")
+    .SetKey("<YOUR_API_KEY>");
+```
+
+{% /section %}
+{% section #step-5 step=5 title="Initialize database" %}
+
+Once the Appwrite Client is initialized, create a function to configure a todo collection.
+
+```kotlin
+val databases = Databases(client)
+
+var todoDatabase: DatabaseModel? = null
+var todoCollection: Collection? = null
+
+fun prepareDatabase() = runBlocking {
+    todoDatabase = databases.create(UUID.randomUUID().toString(), "TodosDB")
+    todoCollection = databases.createCollection(todoDatabase?.id!!, UUID.randomUUID().toString(), "Todos")
+
+    databases.createStringAttribute(todoDatabase?.id!!, todoCollection?.id!!, "title", 255, true)
+    databases.createStringAttribute(todoDatabase?.id!!, todoCollection?.id!!, "description", 255, false, "This is a test description.")
+    databases.createBooleanAttribute(todoDatabase?.id!!, todoCollection?.id!!, "isComplete", true)
+}
+```
+
+{% /section %}
+{% section #step-6 step=6 title="Add documents" %}
+Create a function to add some mock data into your new collection.
+```kotlin
+fun seedDatabase() = runBlocking {
+    val testTodo1 = mapOf(
+        "title" to "Buy apples",
+        "description" to "At least 2KGs",
+        "isComplete" to true
+    )
+
+    val testTodo2 = mapOf(
+        "title" to "Wash the apples",
+        "isComplete" to true
+    )
+
+    val testTodo3 = mapOf(
+        "title" to "Cut the apples",
+        "description" to "Don't forget to pack them in a box",
+        "isComplete" to false
+    )
+
+    databases.createDocument(todoDatabase?.id!!, todoCollection?.id!!, UUID.randomUUID().toString(), testTodo1)
+    databases.createDocument(todoDatabase?.id!!, todoCollection?.id!!, UUID.randomUUID().toString(), testTodo2)
+    databases.createDocument(todoDatabase?.id!!, todoCollection?.id!!, UUID.randomUUID().toString(), testTodo3)
+}
```

Comment:
Same here we should be using `ID.unique()`

Reply:
Oooooops haha